marketplaceItems/tipJar/.gitignore

```diff
@@ -0,0 +1 @@
+entityResources/entity/marketplace/ 
```

Comment:
Oh that was the thing we talked about for partyBall so that I could keep a MP specific JSON in the same place for organization as the JSON file with relative paths